ui/redux/rootReducer.js

```diff
@@ -12,27 +12,21 @@ import modalReducers from './utils/modalReducer'
  */
 
 // actions
+const RECEIVE_DATA = 'RECEIVE_DATA'
```

Comment:
Calling this `RECIEVE_DATA` implies that the only way to update these datasetsByGuid or any xByGuid state trees is to receive data from the server. This seems to be true in general since we always want to save the changes on the server first before updating client state, but just to double check, do you agree that's the case?

Reply:
I agree